components/Tabs/index.tsx

```diff
@@ -12,13 +14,26 @@ import { ResultsPage } from '../ResultsPage'
 export const Tabs: React.FC<ResponseSuccess> = observer((props) => {
   const root: Instance<typeof RootStore> = useStore()
   const tsln = useTranslation<WebTranslations>()
+  const locale = useRouter().locale
 
   return (
     <Tab.Group
       key={root.activeTab}
       defaultIndex={root.activeTab}
       onChange={(index) => {
         root.setActiveTab(index)
+        if (process.browser) {
+          const win = window as Window &
+            typeof globalThis & { adobeDataLayer: any; _satellite: any }
```

Comment:
what the heck

Reply:
Stuffing data into the window object and getting typescript to agree that I am correct! haha